src/main/resources/Development-documentation/gerrit-sceanrios.md

```diff
@@ -0,0 +1,224 @@
+# Gerrit scenarios
+
+These scenarios describes possible use cases that needs consideration during the
+implementation and update of the plugin
+
+<!-- TOC -->
+
+- [Gerrit scenarios](#gerrit-scenarios)
+  - [Create patch set - CASE1](#create-patch-set---case1)
+  - [Create patch set with rebase - CASE2](#create-patch-set-with-rebase---case2)
+  - [Create patch set with merge - CASE3](#create-patch-set-with-merge---case3)
+  - [Create patch set no previous events - CASE4](#create-patch-set-no-previous-events---case4)
+
+<!-- /TOC -->
+
+## Create patch set - CASE1
+
+Preconditions:
+
+- Previous review submits to  the master with hash `C0`
+- The plugin sends SCS with id `E0`
+
+The user does the following:
+
+- Creates a branch
+- Updates the code
+- Squashes the commits to one commits (`C1`)
+- Pushes to `refs/for/[branch name]`
```

Comment:
Since we refer to P1 and P2 etc further down in the code maybe add those here also?

Reply:
Fixed in ee8d791